manager/manager.go

```diff
@@ -75,11 +90,24 @@ func NewManager(opts Options) (*Manager, error) {
 	if opts.WriteKeypair == nil {
 		return nil, errors.New("WriteKeypair must be set")
 	}
+	if opts.MaxRequestsPerVolume == 0 {
+		opts.MaxRequestsPerVolume = 3
+	}
```

Comment:
This is surprising to me. What if I did want this to be 0. Does it make sense for 0 to be an option?

I understand that we can't make use of defaulting in the same way a CLI flag can, and making the type a pointer seems clumsy.

Reply:
0 would imply never leaving a CertificateRequest there at all, e.g. it is deleted immediately after creation/use, which I don't think makes sense (and would take some tweaking of the way cleanup is invoked to make possible).

Perhaps this _is_ something we want to explore in future, but I'd imagine really people want `1` (i.e. only ever have 1 CR for this volume at any given time).

I'm definitely open to expanding the allowed options here if we have a use-case. We should decide whether `0` does make sense as an option, because changing this in future could be painful (as users who don't configure this field explicitly will have a max of 3 enforced, so we won't easily be able to change this to have different behaviour in future without making a breaking Golang API change)

Reply:
If a CertificateRequest *fails* and this is set to zero, would we expect it to be immediately deleted too?